src/controllers/beneficial.owner.gov.controller.ts

```diff
@@ -1,132 +1,26 @@
 import { NextFunction, Request, Response } from "express";
-import { Session } from "@companieshouse/node-session-handler";
 import { logger } from "../utils/logger";
-import { saveAndContinue } from "../utils/save.and.continue";
-import { ApplicationDataType } from "../model";
-import { getFromApplicationData, mapDataObjectToFields, mapFieldsToDataObject, prepareData, removeFromApplicationData, setApplicationData } from "../utils/application.data";
-import {
-  AddressKeys,
-  BeneficialOwnerNoc,
-  HasSamePrincipalAddressKey,
-  ID,
-  InputDateKeys,
-  IsOnSanctionsListKey,
-  NonLegalFirmNoc
-} from "../model/data.types.model";
-import { PrincipalAddressKey, PrincipalAddressKeys, ServiceAddressKey, ServiceAddressKeys } from "../model/address.model";
-import { StartDateKey, StartDateKeys } from "../model/date.model";
-import { BeneficialOwnerGovKey, BeneficialOwnerGovKeys } from "../model/beneficial.owner.gov.model";
 import { BENEFICIAL_OWNER_GOV_PAGE, BENEFICIAL_OWNER_TYPE_URL } from "../config";
-import { v4 as uuidv4 } from "uuid";
+import { getBeneficialOwnerGov, getBeneficialOwnerGovById, postBeneficialOwnerGov, removeBeneficialOwnerGov, updateBeneficialOwnerGov } from "../utils/beneficial.owner.gov";
 
-export const get = (req: Request, res: Response) => {
-  logger.debugRequest(req, `GET ${BENEFICIAL_OWNER_GOV_PAGE}`);
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  logger.debugRequest(req, `${req.method} ${req.route.path}`);
```

Comment:
Is this logger line required? Doesnt it already exist in `getBeneficialOwnerGov`?